### ISSUE TITLE
Profit tracker for Panic Stable

### ISSUE THREAD

**Issue by AuroriSwe** [apex-trader/issue-tracker#135]
Would be awesome to get profit tracking for the Panic Stable option. The bot could track at what price it sold everything to stable and then when it buys back in it could combine them to do the math and calculate the total profit and show this in the closed orders as a "normal" trade.

**Comment by 4n1r0n** [comment]
Basically a stat tracker for panic shorting ;-) I like it. +1

**Comment by joecrypt0** [comment]
Yes. We need this. Currently the panic to stable screws up the stats pretty badly.
